### ISSUE TITLE
Conversation for event based mAP

### ISSUE THREAD

**Issue by ASMIftekhar** [dairui01/Toyota_Smarthome#1]
Hello,
Thanks a lot for the nice work. 

I can see your baseline ADAN model has been evaluated in both frame-based and event based mAP. However, this model predicts per-frame logits. For event based mAP calculation these predictions need to be extended for multiple frames. Can you kindly point out where you do this conversion in the code-base?

**Comment by dairui01** [comment]
I guess what you mean is the PDAN (not ADAN). 
We did not provide the event-based version in our code base. For generating the event-level logits from the model, you need to change the last prediction head of the model to the event-level prediction head (i.e., regression head for predicting the start/end or action duration + a classifier for predicting action category). The training ground truth will be the action event (i.e., the one provided directly in the JSON). Some existing attempts that utilize PDAN for action event prediction can be found [here](https://arxiv.org/pdf/2210.11035.pdf).
